packages/trader/package.json

```diff
@@ -90,7 +90,7 @@
         "@cloudflare/stream-react": "^1.9.1",
         "@deriv-com/analytics": "1.5.9",
         "@deriv-com/quill-tokens": "^2.0.4",
-        "@deriv-com/quill-ui": "1.13.6",
+        "@deriv-com/quill-ui": "^1.13.13",
```

Comment:
Not sure about presence if the caret, I thought we shouldn't add it
https://github.com/binary-com/deriv-app/pull/15828#discussion_r1670271736

Reply:
@kate-deriv merging master still comes with the caret. i'll just keep it the same as master